ex/app/src/main/java/com/example/ex/MainFragment.java

```diff
@@ -0,0 +1,209 @@
+package com.example.ex;
+
+import android.content.Context;
+import android.os.Bundle;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
```

Comment:
After click on submit button it's still possible to press it again and again. Also it's still possible to change rating bars and text while progress is running. And the progress bar itself looks funny, I suggest "replacing" button with round progress bar when the uploading is running and disabling all input fields

[video.mp4.zip](https://github.com/Archi-TV/FlightSubmit/files/5526959/video.mp4.zip)